### DISCUSSION TITLE
Able to get Next.js version from code

### DISCUSSION THREAD

**Discussion by bigint** [vercel/next.js#28952] [Ideas]
### Describe the feature you'd like to request

It would be great if we have the option to get the current next version from the code

### Describe the solution you'd like

Something like

```js
import Version from 'next/version'

<div>v{<Version />}</div>
```

v11.0.0-canary5

### Describe alternatives you've considered

This is how we fetch react version

![image](https://user-images.githubusercontent.com/69431456/132622060-f470fab1-a8a7-4a65-b5dc-f982f0c88a25.png)

**Comment by catnose99** [comment]
You can do the following.

```js
const pkg = require('./package.json'); // depends on where you call this (need to resolve package.json)
const nextVersion = pkg.dependencies.next
```

This works only on server side such as `getServerSideProps` or [API Routes](https://nextjs.org/docs/api-routes/introduction).

**Comment by bigint** [comment]
@catnose99 I'm using `next@canary` version which will not return the exact version.

This is what returned for React's version with `{React.version}`

![image](https://user-images.githubusercontent.com/69431456/132624064-c323f56e-ceaf-4a30-8aa6-f960bf042ae5.png)

**Comment by catnose99** [comment]
How about this

```js
const nextVersion = process.env.npm_package_dependencies_next
```

**Comment by bigint** [comment]
@catnose99 How will that work? without defining the env variable? Thought I tried it and it returns nothin 😢

**Comment by reuixiy** [comment]
Same wishes!
